snow/engine/snowman/transitive.go

```diff
@@ -690,7 +690,7 @@ func (t *Transitive) issueWithAncestors(ctx context.Context, blk snowman.Block)
 	// issue [blk] and its ancestors into consensus
 	status := blk.Status()
 	for status.Fetched() && !t.wasIssued(blk) {
-		err := t.issue(ctx, blk, true)
+		err := t.issue(ctx, t.Ctx.NodeID, blk, true)
```

Comment:
I'm assuming this is only called after `BuildBlock`?

Reply:
Correct - `issueWithAncestors` is only called by `buildBlocks`.